### PR TITLE
Update install.sh to install to /usr/local/bin

### PR DESCRIPTION
**Pull request by joshma**
It's more aggressive than the current strategy, but it means most users
won't have to add to their rc files to get airplane on their paths.

### CODE REVIEW

**Comment by joshma** [comment]
putting review to @colinking given he added it originally

**Comment by joshma** [comment]
heroku also installs to /usr/local/bin https://cli-assets.heroku.com/install.sh
so does railway https://raw.githubusercontent.com/railwayapp/cli/master/install.sh

File: scripts/install.sh [hunk 1]
```diff
@@ -3,28 +3,27 @@
 # Inspired by:
 # - https://github.com/denoland/deno_install/blob/master/install.sh
 #   Copyright 2019 the Deno authors. All rights reserved. MIT license.
-# - https://github.com/superfly/flyctl/blob/master/installers/install.sh
```
Comment (colinking) [inline scripts/install.sh:6]: 🤔 lmao

**Review by colinking — APPROVED** [review]

File: scripts/install.sh (hunk 1 above)
Comment (joshma) [reply scripts/install.sh:6]: Ooh lol I actually thought it was a typo. The Copyright was left for Deno but not fly

Comment (joshma) [reply scripts/install.sh:6]: Will add it back 😊

**Pull request merged**